zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/zeppelinhub/ZeppelinHubRepo.java

```diff
@@ -191,20 +193,45 @@ public void close() {
   @Override
   public Revision checkpoint(String noteId, String checkpointMsg, AuthenticationInfo subject)
       throws IOException {
-    // Auto-generated method stub
-    return null;
+    if (StringUtils.isBlank(noteId)) {
+      return null;
```

Comment:
This might be a subject of another pr because i am following the convention
of the other notebookrepo.

This is out of the scope of this pr.

On Friday, 19 August 2016, Alexander <notifications@github.com> wrote:

> In zeppelin-zengine/src/main/java/org/apache/zeppelin/
> notebook/repo/zeppelinhub/ZeppelinHubRepo.java
> <https://github.com/apache/zeppelin/pull/1338#discussion_r75452980>:
>
> > @@ -191,20 +193,45 @@ public void close() {
> >    @Override
> >    public Revision checkpoint(String noteId, String checkpointMsg, AuthenticationInfo subject)
> >        throws IOException {
> > -    // Auto-generated method stub
> > -    return null;
> > +    if (StringUtils.isBlank(noteId)) {
> > +      return null;
>
> To be consistent, may be we should create Revision.EMPTY or something
> like that?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/zeppelin/pull/1338/files/aef5cf3d36067c1171da078d7df83640b9a060c1#r75452980>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC_n5fApodv8GErmadIab_qSho5bO0vZks5qhXiPgaJpZM4JmJye>
> .
>